### PR TITLE
add slack invite details

### PR DESCRIPTION
**Pull request by markxnelson**
This PR updates the readme to tell users how they can get an invitation to join our public slack channel.

### CODE REVIEW

**Comment by rjeberhard** [comment]
@rosemarymarano, can you please copy edit this?  Thanks!

File: README.md [hunk 1]
```diff
@@ -222,6 +222,10 @@ Replace `NAMESPACE` with the namespace that the operator is running in.
 
 To remove more than one operator, repeat these steps for each operator namespace.
 
+## Need more help?
+
+We have a public slack channel where you can get in touch with us to ask questions about using the operator.  To join our channel, please [visit this site to get an invitation](https://weblogic-slack-inviter.herokuapp.com/).  The invitation email will include details of how to access our slack workspace.  Once you are logged in, please come to #operator and say hello!
```
Comment (rosemarymarano) [inline README.md]: Once you are logged in -> After you are logged in

File: README.md [hunk 2]
```diff
@@ -222,6 +222,10 @@ Replace `NAMESPACE` with the namespace that the operator is running in.
...
+
 # Developer guide
 
 Developers interested in this project are encouraged to read the [Developer guide](site/developer.md) to learn how to build the project, run tests, and so on.  The Developer guide also provides details about the structure of the code, coding standards, and the Asynchronous Call facility used in the code to manage calls to the Kuberentes API.
```
Comment (rosemarymarano) [inline README.md]: typo Kuberentes -> Kubernetes

File: README.md (hunk 1 above)
Comment (rjeberhard) [inline README.md]: Isn't it "Slack"? (capitalized)

Comment (rosemarymarano) [reply README.md]: Yes, I think so too.

**Pull request merged**